app/services/account-service.js

```diff
@@ -306,6 +306,19 @@ const authenticate = async (email, password) => {
   }
   const isUser = await bcrypt.compare(password, user.passwordHash);
   if (isUser) {
+    // assign role on JWT; default to least privilege
```

Comment:
For a handful of roles this is probably fine. But I wouldn't to add many more; if we find we need to add 2 or more I think that would be the occasion to refactor to using a `roles` table with role ids. 

This could simplify the `is_admin`, `is_coordinator` cols on the logins table, too, because the roles table would define jsut what a "coordinator" role is. Make the lookup ask, "do you have the role(s) needed? if not good bye".